apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -8,6 +8,7 @@ import {connect} from 'react-redux';
 import {SignInState} from '@cdo/apps/templates/currentUserRedux';
 import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import InlineMarkdown from '@cdo/apps/templates/InlineMarkdown';
```

Comment:
for my own education, can you help me understand the differences between SafeMarkdown and InlineMarkdown? I see that InlineMarkdown renders into a span rather than a div, but it also looks like any HTML in the input might be handled differently. No need to go down a rabbit hole of research here but I would love to hear whatever you may already know.

Reply:
I think that `InlineMarkdown` only handles very standard markdown syntax (strong tags, italics, links, etc). `SafeMarkdown` on the other hand has support for Blockly xml, the HTML details element, and other "fancier" things. Notably, most, if not all, of the extra supported elements of `SafeMarkdown` don't actually make sense in an inline context anyway.